src/main/target/KAKUTEF7/target.c

```diff
@@ -28,7 +28,7 @@
 #include "drivers/timer_def.h"
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
-    DEF_TIM(TIM1, CH3, PE13, TIM_USE_PPM,   0, 1), // PPM , DMA2_ST6
+    DEF_TIM(TIM1, CH3, PE13, TIM_USE_ANY,   0, 1), // CAM_CONTROL , DMA2_ST6
```

Comment:
Shouldn't make a difference, PPM will only be assigned if PPM is enabled, so I would probably leave that in.